src/components/ContentNode.vue

```diff
@@ -115,17 +122,22 @@ function renderNode(createElement, references) {
   ));
 
   const renderTableCell = (
-    element, attrs, data, cellIndex, rowIndex, extendedData,
+    element, attrs, data, cellIndex, rowIndex, extendedData, alignments,
   ) => {
     const { colspan, rowspan } = extendedData[`${rowIndex}_${cellIndex}`] || {};
     // if either is `0`, then its spanned over and should not be rendered
     if (colspan === 0 || rowspan === 0) return null;
-    return createElement(element, { attrs: { ...attrs, colspan, rowspan } }, (
+    const cellAttrs = attrs;
+    const align = alignments[cellIndex] || TableColumnAlignments.unset;
+    if (align !== TableColumnAlignments.unset) cellAttrs.class = `${align}-cell`;
+    return createElement(element, { attrs: { ...cellAttrs, colspan, rowspan } }, (
```

Comment:
I think classes have their own dedicated key they should be passed through - https://v2.vuejs.org/v2/guide/render-function.html#The-Data-Object-In-Depth

```suggestion
    const align = alignments[cellIndex] || TableColumnAlignments.unset;
    let classes = null;
    if (align !== TableColumnAlignments.unset) classes = `${align}-cell`;
    return createElement(element, { attrs: { ...attrs, colspan, rowspan }, class: classes }, (
```

Reply:
This looks nicer! I was wondering if there was a specific property for CSS classes, but i didn't dig deep enough to see it. I've pushed up this change. Thanks!